tests/unit/test_clidocs.py

```diff
@@ -72,3 +75,128 @@ def test_handle_recursive_output(self):
         self.help_command.obj = operation
         self.operation_handler.doc_output(self.help_command, 'event-name')
         self.assert_rendered_docs_contain('( ... recursive ... )')
+
+
+class TestTopicListerDocumentEventHandler(unittest.TestCase):
+    def setUp(self):
+        self.session = mock.Mock()
+        self.file_creator = FileCreator()
+        self.descriptions = [
+            'This describes the first topic',
+            'This describes the second topic'
+        ]
+        self.tags_dict = {
+            'topic-name-1': {
+                'title': ['The first topic title'],
+                'description': [self.descriptions[0]],
+                'category': ['General Topics', 'Troubleshooting']
+            },
+            'topic-name-2': {
+                'title': ['The second topic title'],
+                'description': [self.descriptions[1]],
+                'category': ['General Topics']
+            }
+        }
+        self.topic_tag_db = TopicTagDB(self.tags_dict)
+        self.cmd = TopicListerCommand(self.session, self.topic_tag_db)
+        self.doc_handler = TopicListerDocumentEventHandler(self.cmd)
+
+    def tearDown(self):
+        self.file_creator.remove_all()
+
+    def test_breadcrumbs(self):
+        self.doc_handler.doc_breadcrumbs(self.cmd)
+        self.assertEqual(self.cmd.doc.getvalue().decode('utf-8'), '')
+        self.cmd.doc.target = 'html'
+        self.doc_handler.doc_breadcrumbs(self.cmd)
+        self.assertEqual(
+            '[ :doc:`aws <../reference/index>` ]',
+            self.cmd.doc.getvalue().decode('utf-8')
+        )
+
+    def test_title(self):
+        self.doc_handler.doc_title(self.cmd)
+        self.assertIn(self.cmd.title, self.cmd.doc.getvalue().decode('utf-8'))
+
+    def test_description(self):
+        self.doc_handler.doc_description(self.cmd)
+        self.assertIn(
+            self.cmd.description,
+            self.cmd.doc.getvalue().decode('utf-8')
+        )
+
+    def _assert_categories_and_topics(self, contents):
+        for category in self.cmd.categories:
+            self.assertIn(category, contents)
+        for entry in self.cmd.entries:
+            self.assertIn('* '+self.cmd.entries[entry], contents)
```

Comment:
I think the [plus sign should have spaces on either side](https://www.python.org/dev/peps/pep-0008/#other-recommendations).